### DISCUSSION TITLE
Multiple Kirby instances

### DISCUSSION THREAD

**Discussion by Jorckx** [johannschopplich/nuxt-kql#15] [Q&A]
hi!

For a personal project I want to fetch content from another kirby-instance that I've setup as a headless CMS. In the docs I can only find the documentation to configure 1 baseurl. It would be cool if I could fetch from another Kirby cms for a another nuxt-page. is this possible with nuxt-kql without manually setting up the fetching via vanilla fetch for that one speciphic page?

**Comment by johannschopplich** [comment]
Hi there!

Great idea. At the moment, Nuxt KQL strictly works with one Kirby instance. Because Nuxt KQL provides a server route to proxy the request and mask the Kirby URL, not only the frontend, but the backend code has to be touched as well. If you don't need to hide the credentials to your server, a plain fetch request might be the easiest option (you can put that into a server route as well).

Although this is possible, I don't have a use-case for it right now and probably won't implement it in the near future. If you require this feature, feel free to sponsor the development of this request. 🙂
